cmd/agent/check.go

```diff
@@ -181,6 +193,13 @@ const (
 	dashboardType = "dashboard"
 	redisType     = "redis"
 	qdbType       = "qdb"
+
+	addrPathType  = "addr"
+	idPathType    = "id"
+	otherPathType = "other"
```

Comment:
what's addrPathType? idPathType? otherPathType?

I think we can use log for log, data for persistent data and proc for current running process data.